internal/impl/pure/scanner_json.go

```diff
@@ -0,0 +1,94 @@
+package pure
+
+import (
+	"context"
+	json "encoding/json"
+	"errors"
+	"io"
+
+	"github.com/benthosdev/benthos/v4/public/service"
+)
+
+const (
+	sjsonFieldContinueOnError = "continue_on_error"
+)
+
+func jsonDocumentScannerSpec() *service.ConfigSpec {
+	return service.NewConfigSpec().
+		Stable().
+		Summary("Consumes JSON documents line-by-line.").
```

Comment:
The decoder doesn't require line breaks so this should be something like "Consumes a stream of one or more JSON documents".